po/fr.po

```diff
@@ -57,86 +54,75 @@ msgstr "Suppression d’anciennes sauvegardes"
 
 #: src/backup_folder.vala:364 src/backup_rsync.vala:377
-#, fuzzy
 msgid "Asked for freeing disk space when there is free space. Aborting backup"
 msgstr ""
 "Impossible de libérer de l’espace disque pour démarrer une sauvegarde. "
-"Interruption de la sauvegarde.\n"
+"Interruption de la sauvegarde."
 
 #: src/backup_folder.vala:379 src/backup_rsync.vala:392
 msgid "Backup aborted"
 msgstr "Sauvegarde abandonnée"
 
 #: src/backup_folder.vala:406 src/backup_rsync.vala:419
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to delete folders: %s"
-msgstr ""
-"Impossible de restaurer le fichier\n"
-"\n"
-"%s"
+msgstr "Impossible de restaurer le fichier %s"
 
 #: src/backup_folder.vala:422 src/backup_rsync.vala:435
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to delete aborted backups: %s"
-msgstr ""
-"Impossible de libérer de l’espace disque pour démarrer une sauvegarde. "
-"Interruption de la sauvegarde.\n"
+msgstr "Impossible de supprimer la sauvegarde échouée : %s."
 
 #: src/backup_folder.vala:447 src/backup_rsync.vala:460
-#, fuzzy, c-format
+#, c-format
 msgid "Backing up folder %s"
-msgstr "Sauvegarde du dossier %s\n"
+msgstr "Sauvegarde du dossier %s"
 
 #: src/backup_folder.vala:468 src/backup_rsync.vala:481
-#, fuzzy, c-format
+#, c-format
 msgid "Excluding folder %s"
-msgstr "Exclusion du dossier %s.\n"
+msgstr "Exclusion du dossier %s."
 
 #: src/backup_folder.vala:477 src/backup_rsync.vala:490
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to launch rsync for '%s'. Aborting backup"
-msgstr ""
-"Impossible de libérer de l’espace disque pour démarrer une sauvegarde. "
-"Interruption de la sauvegarde.\n"
+msgstr "Impossible de lancer rsync pour \"%s\". Sauvegarde abandonnée."
 
 #: src/backup_folder.vala:496 src/backup_rsync.vala:509
 #, c-format
 msgid "There was a problem when backing up the folder '%s'"
-msgstr ""
+msgstr "Problème en sauvegardant le dossier '%s'"
 
 #: src/backup_folder.vala:511 src/backup_rsync.vala:524
-#, fuzzy, c-format
+#, c-format
 msgid "Backing up %s"
-msgstr "Sauvegarde du dossier %s\n"
+msgstr "Sauvegarde du dossier %s"
 
 #: src/backup_folder.vala:544 src/backup_folder.vala:545
 #: src/backup_rsync.vala:557 src/backup_rsync.vala:558
-#, fuzzy
 msgid "Syncing disk"
-msgstr "Synchronisation du disque\n"
+msgstr "Synchronisation du disque"
 
 #: src/backup_folder.vala:551 src/backup_rsync.vala:564
 msgid "Failed to launch sync command"
-msgstr ""
+msgstr "Échec du chargement de la commande sync"
 
 #: src/backup_folder.vala:574 src/backup_rsync.vala:587
-#, fuzzy
 msgid "Failed to rename backup folder. Aborting backup"
 msgstr ""
-"Impossible de libérer de l’espace disque pour démarrer une sauvegarde. "
-"Interruption de la sauvegarde.\n"
+"Impossible de renommer le dossier de sauvegarde. Interruption de la "
+"sauvegarde."
 
 #: src/backup_folder.vala:587 src/backup_rsync.vala:600
 msgid "Failed to launch final sync command"
-msgstr ""
+msgstr "Échec du chargement de la commande sync"
```

Comment:
This one is not the proper translation ... Even literally ...

Reply:
I din't understand in english what does it mean.